File: src/test/java/com/seaweed/identifier/IdentifierApplicationTests.java
package com.seaweed.identifier;

import com.seaweed.identifier.user.UserService;
import org.junit.jupiter.api.Test;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.test.context.SpringBootTest;
import org.springframework.data.domain.PageRequest;
import org.springframework.data.domain.Pageable;


@SpringBootTest
class IdentifierApplicationTests {

	@Autowired
	private UserService userService;

	@Test
	void contextLoads() {
		Pageable pageReq = PageRequest.of(0,1);
		System.out.println(userService.findUsers(pageReq).toString());
	}
}

File: src/test/kotlin/com/seaweed/identifier/IdentifierApplicationTests.kt
package com.seaweed.identifier

import com.seaweed.identifier.user.UserService
import org.junit.jupiter.api.Test
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.boot.test.context.SpringBootTest

@SpringBootTest
class IdentifierApplicationTests {

	@Autowired
	private lateinit var userService : UserService

	@Test
	fun contextLoads() {
		userService.findAllUsers().forEach{i-> println(i.toString()) }
	}

	@Test
	fun createTest(){
		userService.saveUser("홍길동","test");
	}

	@Test
	fun updateTest(){
		userService.updateNameById("test","홍길동");
	}

}
